pkg/operator/k8sutil/pod.go

```diff
@@ -119,6 +120,43 @@ func GetSpecContainerImage(spec v1.PodSpec, name string, initContainer bool) (st
 	return image.Image, nil
 }
 
+// Replaces the pod default toleration of 300s used when the node controller
+// detect a not ready node (node.kubernetes.io/unreachable)
+func AddUnreachableNodeToleration(podSpec *v1.PodSpec) {
+	// The amount of time for this pod toleration can be modified by users
+	// changing the value of <ROOK_UNREACHABLE_NODE_TOLERATION_LAPSE> Rook operator
+	// variable.
+	// Node controller will wait 40 seconds by default before mark a node as
+	// unreachable. After 40s + ROOK_UNREACHABLE_NODE_TOLERATION_LAPSE the pod
+	// will be scheduled in other node
+	// Only one <toleration> to <unreachable> nodes can be added
+	var tolerationSeconds int64 = 5
+	urTolerationDuration := os.Getenv("ROOK_UNREACHABLE_NODE_TOLERATION_LAPSE")
+	if urTolerationDuration != "" {
+		if duration, err := time.ParseDuration(urTolerationDuration); err != nil {
+			logger.Warningf("using default value for <node.kubernetes.io/unreachable> toleration: %v seconds", tolerationSeconds)
```

Comment:
```suggestion
			logger.Warningf("using default value for <node.kubernetes.io/unreachable> toleration %s", time.Duration(tolerationSeconds))
```

Reply:
I prefer to use the  variable "tolerationSeconds" to store seconds initialized to 5 (the default) delay) and it is int64 because it is needed a pointer to this type in the v1.Toleration.  
If used as you suggest it will print "... 5ns", and ye, i can multiply to get seconds ... but i think is more readable in this way.

Reply:
Faire enough.